source/docs/prismic/setup.md

```diff
@@ -0,0 +1,158 @@
+---
+id: prismic-setup
+title: Setup
+---
+
+Here are the steps to install and configure the Front-Commerce Prismic module. This guide assumes that you are working in [a Front-Commerce skeleton environment](https://gitlab.com/front-commerce/front-commerce-skeleton/) and you have access to a Prismic repository.
+
+## Install the module with npm:
+
+First, make sure you have access to [the Front-Commerce Prismic module repository](https://gitlab.com/front-commerce/front-commerce-prismic/). Then you can run the following command:
+
+```sh
+npm i git+ssh://git@gitlab.com/front-commerce/front-commerce-prismic.git
```

Comment:
While the shorthand may leak during support or internal discussion, I think that it is better to be as explicit as possible in the documentation:
```suggestion
npm install git+ssh://git@gitlab.com/front-commerce/front-commerce-prismic.git
```